apollo-runtime/src/main/java/com/apollographql/android/RealApolloCall.java

```diff
@@ -126,6 +126,7 @@ private Request httpRequest(Operation operation) throws IOException {
         .post(requestBody)
         .header("Accept", ACCEPT_TYPE)
         .header("Content-Type", CONTENT_TYPE)
+        .header(HttpCacheInterceptor.CACHE_CONTROL_HEADER, "cache")
```

Comment:
for now all request are cached, we need specs for our custom cache control header